src/index.js

```diff
@@ -1,3 +1,40 @@
+import {
```

Comment:
All this import/export logic should go into the `CreditCardDetails` main module. Will move it in the adjustments branch.